libobs/CMakeLists.txt

```diff
@@ -66,7 +63,56 @@ target_sources(
           obs-video.c
           obs-video-gpu-encode.c
           obs-view.c
-          obs-config.h)
+          $<$<BOOL:${ENABLE_HEVC}>:obs-hevc.c>
+          $<$<BOOL:${ENABLE_HEVC}>:obs-hevc.h>)
+
+target_sources(
+  libobs
+  PRIVATE util/array-serializer.c
+          util/array-serializer.h
+          util/base.c
+          util/base.h
+          util/bitstream.c
+          util/bitstream.h
+          util/bmem.h
+          util/bmem.c
+          util/c99defs.h
+          util/cf-lexer.c
+          util/cf-lexer.h
+          util/cf-parser.c
+          util/cf-parser.h
+          util/config-file.c
+          util/circlebuf.h
+          util/config-file.h
+          util/crc32.c
+          util/crc32.h
+          util/darray.h
+          util/dstr.c
+          util/dstr.h
+          util/file-serializer.c
+          util/file-serializer.h
+          util/lexer.c
+          util/lexer.h
+          util/pipe.h
+          util/platform.c
+          util/platform.h
+          util/profiler.c
+          util/profiler.h
+          util/profiler.hpp
+          util/serializer.h
+          util/sse-intrin.h
+          util/task.c
+          util/task.h
+          util/text-lookup.c
+          util/text-lookup.h
+          util/threading.h
+          util/utf8.c
+          util/utf8.h
+          util/util.hpp
+          util/uthash.h
+          util/util_uint64.h
+          util/util_uint128.h
+          util/curl/curl-helper.h)
```

Comment:
Sort.

Reply:
Apart from the config-file, everything is sorted:

* Alphabetically per directory scope
* Subdirectories after files

Reply:
The stray config-file / circlebuf mixup was mostly what I was referring to, yes.